### ISSUE TITLE
Deprecation warnings under Rails 5.2.4.3

### ISSUE THREAD

**Issue by nfm** [kickstarter/rack-attack#484]
Rails 5.2.4.3 was just released, including a fix for [CVE-2020-8165](https://groups.google.com/forum/#!topic/rubyonrails-security/bv6fW4S0Y1c) (Potentially unintended unmarshalling of user-provided objects in MemCacheStore and RedisCacheStore).

After upgrading to Rails 5.2.4.3, I'm getting a deprecation warning that backtraces to `rack-attack`. The warning is:

```
Using a different value for the raw option when reading and writing
to a cache key is deprecated for :redis_cache_store and Rails 6.0
will stop automatically detecting the format when reading to avoid
marshal loading untrusted raw strings.
```

This is in [redis_cache_store.rb](https://github.com/rails/rails/blob/abd27d5121573ab591d1501afa5f81e2e9da8c90/activesupport/lib/active_support/cache/redis_cache_store.rb#L424).

It looks like https://github.com/rails/rails/commit/467e3399c9007996c03ffe3212689d48dd25ae99 (which addresses the CVE) introduced this deprecation warning.

From what I can tell, it's because `rack-attack` is [writing to the store with raw: true](https://github.com/kickstarter/rack-attack/blob/d92f66c8d9fd4ff593e4bc3db7d5f9b82f605b6a/lib/rack/attack/store_proxy/redis_cache_store_proxy.rb#L28), then later [reading from the store _without_ raw: true](https://github.com/kickstarter/rack-attack/blob/d92f66c8d9fd4ff593e4bc3db7d5f9b82f605b6a/lib/rack/attack/store_proxy/redis_cache_store_proxy.rb#L18) - although I may be mistaken.

**Comment by grzuy** [comment]
Thanks for reporting!

**Comment by grzuy** [comment]
Fix released in `v6.3.1`.
